python/ray/data/_internal/stats.py

```diff
@@ -1173,11 +1173,13 @@ def to_string(self) -> str:
         ):
             out += "\nDataset iterator time breakdown:\n"
             if self.block_time.get():
-                out += "* Total time user code is blocked: {}\n".format(
-                    fmt(self.block_time.get())
+                out += (
+                    "* Total time user thread is blocked by iter_batches: {}\n".format(
+                        fmt(self.block_time.get())
```

Comment:
let's also rename the `self.block_time` and `self.user_time` to a slightly more descriptive name as well